lua/telescope/builtin/__lsp.lua

```diff
@@ -177,24 +177,36 @@ local function list_or_jump(action, title, funname, params, opts)
   opts.reuse_win = vim.F.if_nil(opts.reuse_win, false)
   opts.curr_filepath = vim.api.nvim_buf_get_name(opts.bufnr)
 
-  vim.lsp.buf_request(opts.bufnr, action, params, function(err, result, ctx, _)
-    if err then
-      vim.api.nvim_err_writeln("Error when executing " .. action .. " : " .. err.message)
-      return
-    end
+  vim.lsp.buf_request_all(opts.bufnr, action, params, function(results_per_client)
+    local items = {}
+    local first_encoding
+
+    for client_id, result_or_error in pairs(results_per_client) do
+      local error, result = result_or_error.error, result_or_error.result
+      if error then
+        vim.api.nvim_err_writeln("Error when executing " .. action .. " : " .. error.message)
+        return
+      end
```

Comment:
I'm contemplating whether we should collect all the errors first, then raise them later on.

Like suppose some lsp server returns an error but another returns valid results. In this case, I'm thinking it might be more user friendly to still let telescope handle the rest of the results, and then show a warning for the error.
If there is only errors, we can throw an error.

Reply:
gotcha, I thought about it while changing the code but then I looked to see if lua had a `next` or `continue` instruction. And since I didn't find it, I moved on

I'll give it a stab